### PR TITLE
Improve and synchronize radio + checkbox styles

### PR DESCRIPTION
**Pull request by busheikin**
# Link to Github Issue

https://github.com/AmpleOrganics/Blaze.vue/issues/232#event-2181567483

# Description

- Fixes checkbox docs input
- extracts shared radio + checkbox css into a mixin
- improves and synchronizes styles for radio and checkbox

### CODE REVIEW

File: src/assets/styles/mixins/shared-checkbox-styles.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// Mixin for shared input styles
+
+@mixin shared-checkbox-styles {
+  .ao-checkbox {
```
Comment (ZainW) [inline src/assets/styles/mixins/shared-checkbox-styles.scss]: there is a pr up to change the classname from ao-checkbox to ao-radio for radio buttons,  so we need to be wary of that change

Comment (ZainW) [reply src/assets/styles/mixins/shared-checkbox-styles.scss]: https://github.com/AmpleOrganics/Blaze.vue/pull/302

Comment (busheikin) [reply src/assets/styles/mixins/shared-checkbox-styles.scss]: cool, see my new commit

**Comment by ZainW** [comment]
rebase/fix merge conflicts please, then ill review and ask for another

**Comment by busheikin** [comment]
Ready for review

File: src/components/AoRadio.vue [hunk 2]
```diff
@@ -5,11 +5,14 @@
       v-bind="$attrs"
       :value="val"
       type="radio"
+      class="ao-checkbox__input"
```
Comment (Boshes) [inline src/components/AoRadio.vue]: should this class and the other one be ao-radio__x since the component is radio

Comment (ZainW) [reply src/components/AoRadio.vue]: yup,
 ill do the fixups for this PR

Comment (ZainW) [reply src/components/AoRadio.vue]: fixed

**Pull request merged**